### PR TITLE
Rewrote description of VIC-IV "hot" registers

### PR DESCRIPTION
**Pull request by dansanderson**
This rewrites description of VIC-IV "hot" registers for clarity, based on recent Discord discussion. See issue #477.

### CODE REVIEW

**Comment by gurcei** [comment]
Will just paste the comment from the issue here too. Happy to merge as-is, or if you want to try add the suggestions in the comment below, we can hold off till then:
------
Yes this is looking much better than what was there already to me. Maybe I'm happy for this term "precise registers" to still get a mention somewhere though, as there are other locations within the manual that do use the "precise" term (so I think it has more traction than "vernier").

I liked your list of affected vic-ii registers, I wonder if it's worth putting them in a table so that we can clarify which vic-iv registers they map to (in the "hot register" sense)

**Comment by dansanderson** [comment]
"precise" is only used two other times in the chipset reference and only to refer to relocation registers that allow 24-bit byte addresses. So they're an example of a purpose of hot registers (VIC-II page addresses vs. VIC-IV byte addresses) but not the only purpose.

A table describing the complete hot register behavior would be nice, but I'd need some help figuring out and confirming the data. I can make some guesses based on the register descriptions. These are mostly video mode registers.

D011: RB8, ECM, BMM, BLNK, RSEL, YSCL (3)
D016: RST, MCM, CSEL, XSCL (3)
D018: VS (4), CB (3)
D031 (VIC-III): H640, FAST, ATTR, BPM, V400, H1280, MONO, INT
DD00: "[bank bits](https://www.c64-wiki.com/wiki/VIC_bank)" (2)

**Comment by gurcei** [comment]
Yep, two other occurances of 'precise' in the manual in relation to this, but also I think having that convo with LGB and seeing how he also used the term 'precise' as well added to my impression that the term was getting traction, in the spoken form. So given that, maybe the term 'precise' could propogate further?

Aah, or was your impression that other hot-registers weren't necessarily related to vic-ii (less-precise) versus vic-iv (more-precise)? Ok, if so, then get where you're coming from on this.

Gotta admit I haven't explored all hot registers relationships, so maybe it's fine to just flesh out the ones you've grasped into this table, and maybe we can consider any other less-apparent hot registers later on?

**Comment by gurcei** [comment]
Okidoke, thx for the clarification. Ok, happy to stick with your current text then :-)

**Review by gurcei — APPROVED** [review]
Merging in, thx :-)

**Pull request merged**
